File: 1440-convert-integer-to-the-sum-of-two-no-zero-integers/1440-convert-integer-to-the-sum-of-two-no-zero-integers.java
class Solution {
    public boolean isValid(int n){
        String s = String.valueOf(n);
        return !s.contains("0");
    }
    public int[] getNoZeroIntegers(int n) {
        for(int i=1;i<n+1;i++){
            if(isValid(i) && isValid(n-i)){
                return new int[]{i,n-i};
            } 
        }
        return new int[]{};
    }
}

File: 1440-convert-integer-to-the-sum-of-two-no-zero-integers/1440-convert-integer-to-the-sum-of-two-no-zero-integers.py
class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        def isValid(num):
            #print(num)
            return '0' not in str(num)
        for i in range(1,n+1):
            if isValid(i) and isValid(n-i):
                return [i,n-i]
            
